src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -108,7 +108,7 @@ public function requireOnce($file)
      */
     public function put($path, $contents, $lock = false)
     {
-        return file_put_contents($path, $contents, $lock ? LOCK_EX : 0);
+        return file_put_contents($path, $contents, $lock ? LOCK_SH : 0);
```

Comment:
According to [php manual](https://secure.php.net/manual/en/function.file-put-contents.php), file_put_contents only support LOCK_EX lock.